docs/pages/api-docs/data-grid/grid-api.md

```diff
@@ -0,0 +1,101 @@
+# GridApi Interface
+
+<p class="description">The full grid API.</p>
+
+## Import
+
+```js
+import { GridApi } from '@material-ui/x-grid';
+// or
+import { GridApi } from '@material-ui/data-grid';
+```
+
+## Properties
+
+| Name                                                                                            | Type                                                                                                                                                                                     | Default | Description                                                                                                        |
+| :---------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------ | :----------------------------------------------------------------------------------------------------------------- |
+| <span class="prop-name required">applyFilter<abbr title="required">\*</abbr></span>             | <span class="prop-type">(item: GridFilterItem, linkOperator?: GridLinkOperator) =&gt; void</span>                                                                                        |         |                                                                                                                    |
```

Comment:
For non-props, it feels to see "required". Shouldn't we do the opposite, and label the optional properties?